pom.xml

```diff
@@ -85,12 +91,17 @@
                 <artifactId>coveralls-maven-plugin</artifactId>
                 <version>4.3.0</version>
                 <configuration>
-                    <!-- TODO: Don't commit repoToken pubically! Find a way to integrate securely -->
-                    <repoToken>yourcoverallsprojectrepositorytoken</repoToken>
                     <jacocoReports>
                         <param>${project.reporting.outputDirectory}/jacoco-ut/jacoco.xml</param>
                     </jacocoReports>
                 </configuration>
+                <dependencies>
+                    <dependency>
+                        <groupId>javax.xml.bind</groupId>
+                        <artifactId>jaxb-api</artifactId>
+                        <version>2.3.1</version>
+                    </dependency>
+                </dependencies>
```

Comment:
Added jaxb-api as a workaround for this issue https://github.com/trautonen/coveralls-maven-plugin/issues/112